File: trunk/doc/Examples_SPL/Expresiones/Expresiones/PrintInfix/multInfix.cs
using System;
using Expresiones;


namespace Expresiones
{
    /**
     * Clase que define un entero
     * */
    public class MultInfix : BinaryOperatorInfix, Mult
    {
        ExpressionInfix exp_izquierda;
        ExpressionInfix exp_derecha;
        /**
         * Constructor
         * */
        public MultInfix(ExpressionInfix izq, ExpressionInfix derch)
        {
            this.exp_izquierda = izq;
            this.exp_derecha = derch;
        }
        /**
         * Método que imprime en formato infijo
         * */
        void ExpressionInfix.printInfix()
        {
            Console.Write("(");
            exp_izquierda.printInfix();
            Console.Write("*");
            exp_derecha.printInfix();
            Console.Write(")");
        }
    }
}

File: trunk/doc/Examples_SPL/expressionsCaesarJ/src/printInfix/MultInfix.java
cclass printInfix.PrintInfix;

public cclass MultInfix extends BinaryOperatorInfix & Mult{
	protected ExpressionInfix op1;
	protected ExpressionInfix op2;
	public MultInfix(ExpressionInfix op1,ExpressionInfix op2){
		this.op1=op1;
		this.op2=op2;
	}
	public void print(){
		System.out.print("("+op1+"*"+op2+")");
	}

}